solana/bridge_token_factory/programs/bridge_token_factory/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "bridge_token_factory"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "bridge_token_factory"
+
+[features]
+default = []
+cpi = ["no-entrypoint"]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+idl-build = [
+    "anchor-lang/idl-build",
+    "anchor-spl/idl-build",
+    "wormhole-anchor-sdk/idl-build",
+    "wormhole-anchor-sdk/mainnet",
+]
+devnet = ["wormhole-anchor-sdk/solana-devnet"]
+mainnet = ["wormhole-anchor-sdk/mainnet"]
+
+[dependencies]
+anchor-lang = { version = "0.30.1", features = ["init-if-needed"] }
+anchor-spl = { version = "0.30.1", features = ["metadata"] }
+cfg-if = "1.0.0"
+near-sdk = "5.5.0"
```

Comment:
Let's use the borsh crate instead of `near_sdk`, the borsh crate is already imported by the anchor.
```
[[package]]
name = "anchor-lang"
version = "0.30.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "6620c9486d9d36a4389cab5e37dc34a42ed0bfaa62e6a75a2999ce98f8f2e373"
dependencies = [
 "anchor-attribute-access-control",
 "anchor-attribute-account",
 "anchor-attribute-constant",
 "anchor-attribute-error",
 "anchor-attribute-event",
 "anchor-attribute-program",
 "anchor-derive-accounts",
 "anchor-derive-serde",
 "anchor-derive-space",
 "anchor-lang-idl",
 "arrayref",
 "base64 0.21.7",
 "bincode",
 "borsh 0.9.3",
 "bytemuck",
 "getrandom 0.2.15",
 "solana-program",
 "thiserror",
]
```

Reply:
Removed